constraints/miden-vm/chiplets.air

```diff
@@ -0,0 +1,67 @@
+use bitwise::bitwise_chiplet
+use hash::hash_chiplet
+use memory::memory_chiplet
```

Comment:
Based on @grjte's comment in another PR, these should go below the module declaration.